src/main/java/uk/gov/justice/hmpps/prison/aop/connectionproxy/OracleConnectionAspect.java

```diff
@@ -54,11 +59,30 @@ protected Connection openProxySessionIfIdentifiedAuthentication(final Connection
                 log.warn("Failed to set context for proxy user {}", MDC.get(PROXY_USER));
             }
         }
+        else if ("true".equals(MDC.get(SUPPRESS_XTAG_EVENTS))) {
+            assertNotSlow();
+            setContextAuditModuleOnly(pooledConnection);
+        }
```

Comment:
Makes sure the MERGE context is set every time suppression is requested - assuming we are using the primary database.